pixiedust/display/chart/renderers/mapbox/mapBoxMapDisplay.py

```diff
@@ -179,12 +181,31 @@ def renderMapView(self, mbtoken):
 
             # if there's a numeric value field and type is not 'simple', paint the data as a choropleth map
             if self.options.get("kind") and self.options.get("kind").find("simple") < 0 and len(valueFields) > 0:
+                # get value from the "Number of Bins" slider
+                numBins = 5 # default
```

Comment:
Nit pick: You may want to use this more concise syntax which makes it easier to read: 
numBins = self.options.get("numbins", 5)